src/gwt/www/templates/encrypted-sign-in.htm

```diff
@@ -87,6 +87,21 @@ <h1 id="caption_header" class="caption">
                 aria-required="true"
               /><br />
             </p>
+            <p>
```

Comment:
We can't have this extra UI showing by default. It would need to be controlled via a setting, probably in rserver.conf, and either a have new login page for this mode, or perhaps conditionally show this additional UI.

There is an injection mechanism for this page, see logoHtml for an example (corresponding code is in ServerLoginPages.cpp).